pkgs/development/tools/build-managers/sbt/default.nix

```diff
@@ -2,15 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "sbt";
-  version = "1.2.8";
+  version = "1.3.2";
 
   src = fetchurl {
     urls = [
-      "https://dl.bintray.com/sbt/native-packages/sbt/${version}/${pname}-${version}.tgz"
+      "https://piccolo.link/sbt-${version}.tgz",
```

Comment:
This is being used in [https://www.scala-sbt.org/download.html](https://www.scala-sbt.org/download.html)